packages/template-retail-react-app/app/layouts/site-layout-checkout/index.jsx

```diff
@@ -0,0 +1,167 @@
+                    rel="apple-touch-icon"
+                    href={getAssetUrl('static/img/global/apple-touch-icon.png')}
+                />
+                <link rel="manifest" href={getAssetUrl('static/manifest.json')} />
+
+                {/* Urls for all localized versions of this page (including current page)
+                 For more details on hrefLang, see https://developers.google.com/search/docs/advanced/crawling/localized-versions */}
+                {site.l10n?.supportedLocales.map((locale) => (
+                    <link
+                        rel="alternate"
+                        hrefLang={locale.id.toLowerCase()}
+                        href={`${appOrigin}${getPathWithLocale(locale.id, {
+                            location
+                        })}`}
+                        key={locale.id}
+                    />
+                ))}
+                {/* A general locale as fallback. For example: "en" if default locale is "en-GB" */}
+                <link
+                    rel="alternate"
+                    hrefLang={site.l10n.defaultLocale.slice(0, 2)}
+                    href={`${appOrigin}${getPathWithLocale(site.l10n.defaultLocale, {
+                        location
+                    })}`}
+                />
+                {/* A wider fallback for user locales that the app does not support */}
+                <link rel="alternate" hrefLang="x-default" href={`${appOrigin}/`} />
+            </Seo>
+
+            <ScrollToTop />
+
+            <Box id="app" display="flex" flexDirection="column" flex={1}>
+                <SkipNavLink zIndex="skipLink">Skip to Content</SkipNavLink>
+
+                <Box {...styles.headerWrapper}>
+                    <CheckoutHeader />
+                </Box>
+
+                {!isOnline && <OfflineBanner />}
+                <AddToCartModalProvider>
+                    <SkipNavContent
+                        style={{
+                            display: 'flex',
+                            flexDirection: 'column',
+                            flex: 1,
+                            outline: 0
+                        }}
+                    >
+                        <Box
+                            as="main"
+                            id="app-main"
+                            role="main"
+                            display="flex"
+                            flexDirection="column"
+                            flex="1"
+                        >
+                            <OfflineBoundary isOnline={false}>{children}</OfflineBoundary>
+                        </Box>
+                    </SkipNavContent>
+
+                    <CheckoutFooter />
+
+                    <AuthModal {...authModal} />
+                </AddToCartModalProvider>
+            </Box>
+        </>
+    )
+}
+
+SiteLayoutCheckout.propTypes = {
+    children: PropTypes.node
+}
+
+export const getLayout = (page) => {
+    return <SiteLayoutCheckout>{page}</SiteLayoutCheckout>
```

Comment:
To reduce the code duplicate found in SiteLayout and SiteLayoutCheckout, we can reuse the SiteLayout and pass in the header and footer props like this:

```jsx
// In SiteLayoutCheckout
export const getLayout = (page) => {
    return (
        <SiteLayout header={<CheckoutHeader />} footer={<CheckoutFooter />}>
            {page}
        </SiteLayout>
    )
}

```

And in the SiteLayout definition itself, it allows for the header and footer props. And these props are by default set to the default site header and footer.

```jsx
const SiteLayout = ({header = <Header />, footer = <Footer />, children}) => {
...
}
```